src/rules/__tests__/a11y-remove-disable-tooltip.test.js

```diff
@@ -0,0 +1,41 @@
+'use strict'
+
+const {RuleTester} = require('eslint')
+const rule = require('../a11y-remove-disable-tooltip')
+
+const ruleTester = new RuleTester({
+  parserOptions: {
+    ecmaVersion: 'latest',
+    sourceType: 'module',
+    ecmaFeatures: {
+      jsx: true,
+    },
+  },
+})
+
+ruleTester.run('a11y-remove-disable-tooltip', rule, {
```

Comment:
I am curious if there any components that render `IconButton` under the hood (e.g. `SomeComponent.SomeButton`), and whether it's possible to detect these?

Reply:
For Primer components that are using IconButton under the hood, the hope is to enable their the tooltips by default once we flip the value of `unsafeDisableTooltip` to be `false` by default. We will make sure there is no regression or integration issues at dotcom when we enable the tooltips by default in these components and if we end up having issues, we will track them in an issue. Does that answer your question? 🤔